pkg/agent/openflow/client_test.go

```diff
@@ -813,6 +813,78 @@ func Test_client_InstallPodFlows(t *testing.T) {
 	}
 }
 
+func Test_client_UpdatePodFlows(t *testing.T) {
+	podIPv4 := net.ParseIP("10.10.0.66")
+	podIPv4Updated := net.ParseIP("10.10.0.88")
+	podMAC, _ := net.ParseMAC("00:00:10:10:00:66")
+	podOfPort := uint32(100)
+	testCases := []struct {
+		name                   string
+		enableIPv4             bool
+		enableIPv6             bool
+		clientOptions          []clientOptionsFn
+		podInterfaceIPs        []net.IP
+		podUpdatedInterfaceIPs []net.IP
+		vlanID                 uint16
+		trafficEncapMode       config.TrafficEncapModeType
+		expectedFlows          []string
+		expectedNewFlows       []string
+	}{
+		{
+			name:                   "IPv4",
+			enableIPv4:             true,
+			podInterfaceIPs:        []net.IP{podIPv4},
+			podUpdatedInterfaceIPs: []net.IP{podIPv4Updated},
+			expectedFlows: []string{
+				"cookie=0x1010000000000, table=ARPSpoofGuard, priority=200,arp,in_port=100,arp_spa=10.10.0.66,arp_sha=00:00:10:10:00:66 actions=goto_table:ARPResponder",
+				"cookie=0x1010000000000, table=Classifier, priority=190,in_port=100 actions=set_field:0x3/0xf->reg0,goto_table:SpoofGuard",
+				"cookie=0x1010000000000, table=SpoofGuard, priority=200,ip,in_port=100,dl_src=00:00:10:10:00:66,nw_src=10.10.0.66 actions=goto_table:UnSNAT",
+				"cookie=0x1010000000000, table=L3Forwarding, priority=200,ip,reg0=0x200/0x200,nw_dst=10.10.0.66 actions=set_field:0a:00:00:00:00:01->eth_src,set_field:00:00:10:10:00:66->eth_dst,goto_table:L3DecTTL",
+				"cookie=0x1010000000000, table=L2ForwardingCalc, priority=200,dl_dst=00:00:10:10:00:66 actions=set_field:0x64->reg1,set_field:0x100/0x100->reg0,goto_table:IngressSecurityClassifier",
+			},
+			expectedNewFlows: []string{
+				"cookie=0x1010000000000, table=ARPSpoofGuard, priority=200,arp,in_port=100,arp_spa=10.10.0.88,arp_sha=00:00:10:10:00:66 actions=goto_table:ARPResponder",
+				"cookie=0x1010000000000, table=Classifier, priority=190,in_port=100 actions=set_field:0x3/0xf->reg0,goto_table:SpoofGuard",
+				"cookie=0x1010000000000, table=SpoofGuard, priority=200,ip,in_port=100,dl_src=00:00:10:10:00:66,nw_src=10.10.0.88 actions=goto_table:UnSNAT",
+				"cookie=0x1010000000000, table=L3Forwarding, priority=200,ip,reg0=0x200/0x200,nw_dst=10.10.0.88 actions=set_field:0a:00:00:00:00:01->eth_src,set_field:00:00:10:10:00:66->eth_dst,goto_table:L3DecTTL",
+				"cookie=0x1010000000000, table=L2ForwardingCalc, priority=200,dl_dst=00:00:10:10:00:66 actions=set_field:0x64->reg1,set_field:0x100/0x100->reg0,goto_table:IngressSecurityClassifier",
+			},
+		},
+	}
+	for _, tc := range testCases {
```

Comment:
Since only a test case, how about removing the loop? Or add a IPv6 case?

Reply:
+1, or you may remove unused field like `enableIPv6` if there is no meaning to add a specific case for IPv6.

Reply:
Done